P5/Source/Guidelines/en/AI-AnalyticMechanisms.xml

```diff
@@ -481,6 +493,7 @@ This matter is taken up in detail in section <ptr target="#AILA"/>.
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/pc.xml"/>
 </specGrp>
 <specGrp xml:id="DAILA" n="Linguistic Word-level Attributes">
+<include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/att.normalized.xml"/>
```

Comment:
After council f2f discussion we'd suggest to name the class att.lexicographic.normalized to make it very clear that these attributes are only designed for a very particular purpose.